shells/lib/env/env-base.js

```diff
@@ -9,6 +10,9 @@ export class EnvBase {
     this.rootPath = rootPath;
     this.loaderKind = loaderKind;
     this.pathMap = this.createPathMap(rootPath);
+    this._loader = new (this.loaderKind)(this.pathMap);
+    // TODO. Temporary until ArcRunner carries this info.
+    Runtime.setLoader(this._loader);
```

Comment:
If the goal is to propagate the env loader into Runtime, this is not the right place to do that IMO.

Reply:
Agreed, but the right place will take more work so this is temporary to unblock Maria, as above.